src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskClaimByIdControllerTest.java

```diff
@@ -102,6 +105,51 @@ public void user_should_claim_task_when_role_assignment_verification_passed() {
         common.cleanUpTask(taskId);
     }
 
+    @Test
+    public void user_should_claim_task_and_claim_action_recorded_in_replicate_db() throws InterruptedException {
```

Comment:
This should not be in here.  Should be in its own class since we are testing replication not the claim endpoint.

Reply:
Moved this test to new class